src/shared/components/DynamicFunctionsList.tsx

```diff
@@ -53,10 +53,12 @@ const DynamicFunctionsList: FC<Props> = ({onSelect}) => {
     }
   }, [hoveredFunction, tooltipPopup, eventSearchTerm])
 
-  const handleSelectItem = useCallback((func: FluxFunction) => {
-    onSelect(getFluxExample(func))
-    event('flux.function.injected', {name: `${func.package}.${func.name}`})
```

Comment:
Also...please make sure Dumplings is on board with removing this event, as they placed it in.

Reply:
the person who might maybe want to use this sometime for something is gone, so feature matched on other side of the event move

Reply:
@blegesse-git added this event metric, as part of a PR specifically for adding metrics. Think we should at least ping and make sure it's ok to remove. 🤷🏼‍♀️